lib/slices/tree.rb

```diff
@@ -39,7 +39,7 @@ module Tree
       field :position, type: Integer
       field :show_in_nav, type: Boolean, default: false
 
-      scope :minimal, ->{ only(%w(_type page_id path external_url name has_content)) }
+      scope :minimal, ->{ only(%w(_id _type page_id path external_url name has_content)) }
```

Comment:
I'm guessing that the previous version of Mongoid automatically included id when using `only`, now it's possible to not include `_id` which breaks `link_to` as there is no id